generator/integration_tests/golden/v1/golden_thing_admin_client.cc

```diff
@@ -57,12 +57,33 @@ GoldenThingAdminClient::CreateDatabase(std::string const& parent, std::string co
   return connection_->CreateDatabase(request);
 }
 
+StatusOr<google::longrunning::Operation>
+GoldenThingAdminClient::CreateDatabase(ExperimentalTag, NoAwaitTag, std::string const& parent, std::string const& create_statement, Options opts) {
```

Comment:
silly nit: should we name these tags and `std::move` them into the connection call?